frontend/src/main/components/Jobs/JobsTable.js

```diff
@@ -1,8 +1,6 @@
 import React from "react";
-import OurTable, {
-  PlaintextColumn,
-  DateColumn,
-} from "main/components/OurTable";
+import OurTable, { DateColumn } from "main/components/OurTable";
+import { Link } from "react-router-dom";
```

Comment:
To support the suggestion later in the code review...

```suggestion
import { Link } from "react-router-dom";
import Plaintext from "main/components/Utils/Plaintext";
```